vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -1,16 +1,15 @@
+# MUST define global variable with "global"
```

Comment:
Maybe we can add a comment to describe the intention of this script, like
This script is to define **global** variables and please do not add extra biz logic here.

Reply:
Added